packages/Datepicker/src/datepicker.scss

```diff
@@ -117,8 +122,8 @@ $datepicker-width: calc($cell-number * $cell-size - $cell-border - $cell-empty-b
   margin-top: -1px;
 
   &:hover {
-    background: var(--ids-datepicker-content-background-hover);
-    color: var(--ids-datepicker-content-color-hover);
+    @extend %active-date;
```

Comment:
I see you switched the hover colors for the selected one. Is this the desired effect? If so, please remove the variable definitions above.

Reply:
Good point 👍🏻